cogs/modmail.py

```diff
@@ -448,7 +448,7 @@ async def send_scheduled_close_message(self, ctx, after, silent=False):
 
         embed = discord.Embed(
             title="Scheduled close",
-            description=f"This thread will close {silent}{human_delta}.",
+            description=f"This thread will close in {silent}{human_delta}.",
```

Comment:
I think it should be put between `{silent}` and `{human_delta}`. Otherwise if it's silent, the output would be:
`This thread will close in silently 2 hours.`


```suggestion
            description=f"This thread will close {silent}in {human_delta}.",
```